src/server.ts

```diff
@@ -21,8 +21,6 @@ import { VerifierState } from './VerifierState';
 import { generateDid, verifyJws, encryptFor } from './dids';
 import { issuerReducer, prepareSiopRequest, issueVcsToHolder, parseSiopResponse, CredentialGenerationDetals } from './VerifierLogic';
 
-import * as s1 from '@decentralized-identity/sidetree/dist/lib/core/versions/latest/protocol-parameters.json';
```

Comment:
Historically this was needed to ensure that this JSON file was included in the static build. Can you confirm that the build still works for you without this?

Reply:
(I did just confirm though -- builds without this line, so never mind my comment above and feel free to just include this ;-))